.github/workflows/meta_corpus_stats_generator.yml

```diff
@@ -44,27 +44,17 @@ jobs:
           git add -A
           git commit -m "Current version of workflows"
           git push
-      - name: Clone ms3
-        uses: actions/checkout@v2
-        with:
-          repository: johentsch/ms3
-          ref: workflow
-          path: ./ms3
-      - name: Install ms3
-        run: |
-          python -m pip install --upgrade pip
-          python -m pip install -e ./ms3
-      - name: Run ms3 extract
```

Comment:
@johentsch unless I'm awfully wrong, we do need `ms3` because we import `resolve_dir` in line 12 of `update_pages.py`.